download_styling_models.sh

```diff
@@ -1,2 +1,4 @@
-wget https://www.dropbox.com/s/gtwnyp9n49lqs7t/saved-models.zip?dl=1
-unzip saved-models.zip?dl=1
+#!/bin/bash
+wget https://www.dropbox.com/s/gtwnyp9n49lqs7t/saved-models.zip?dl=1 -O saved-models.zip
+unzip saved-models.zip
+rm saved-models.zip
```

Comment:
I think we should avoid deletion of zip file, can come across as a bit intrusive to the user :)